fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java

```diff
@@ -303,11 +349,12 @@ public void finish() throws IOException {
      *
      * @param w Output writer.
      * @param r Zero-based row number.
+     * @param isHidden Whether or not this row is hidden
      * @param row Cells in the row.
      * @throws IOException If an I/O error occurs.
      */
-    private static void writeRow(Writer w, int r, Cell... row) throws IOException {
-        w.append("<row r=\"").append(r + 1).append("\">");
+    private static void writeRow(Writer w, int r, boolean isHidden, Cell... row) throws IOException {
+        w.append("<row r=\"").append(r + 1).append("\" hidden=\"").append(String.valueOf(isHidden)).append("\">");
```

Comment:
Detail: we could make the XML more compact by writing `hidden="true"` when the flag is `true` and nothing otherwise.